css/src/components/popover.scss

```diff
@@ -0,0 +1,36 @@
+/* stylelint-disable scss/at-extend-no-missing-placeholder */
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
+		@extend .button;
```

Comment:
To ensure this component is composable in multiple ways it should not force the presentation of the summary element  to be .button. If the design calls for a button appearance folks can add the .button class to the `<summary>` element.

Reply:
I haven't seen any design specifically for the popover component outside the breadcrumbs design file, and it definitely looks like a button over there. Removing the buttons styles make it look a little off and not obvious that you should click on it, so I'm working on trying to figure out what should I do if it's not a button

Reply:
This one is in discussion with the design team, for now, I've updated the summary a little so it looks like a clickable element, and in combination with the `.button` class, we can get the button look.

Here is a popover design discussion: https://github.com/microsoft/atlas-design/discussions/165